test/recipient-A.sec.pem

```diff
@@ -0,0 +1,3 @@
+-----BEGIN CRYPT4GH PRIVATE KEY-----
+YzRnaC12MQAGc2NyeXB0ABQAAAAALX3Zj05ZwraMXtsGXXURNwARY2hhY2hhMjBfcG9seTEzMDUAPOUGSFOpCTQi5+trEoupv4yJYO4KsaOlJw+I1pU2xqk6M5sbVCWnW/VuNOyODbxKwcOyFopvOZDCN+oU+w==
+-----END CRYPT4GH PRIVATE KEY-----
```

Comment:
In order to minimize alerts from GH about commited private keys these could easily be generated with a script for the tests, or simply be set inside the test files.
Same goes for the encrypted dummy files

Reply:
I will try to see what can be done

Reply:
i put them in the test till we figure out how to generate them.